src/routes/UserSearchContainer.js

```diff
@@ -4,27 +4,22 @@ import {
   get,
   template,
 } from 'lodash';
-import { stripesConnect } from '@folio/stripes/core';
 
+import { stripesConnect } from '@folio/stripes/core';
 import {
   makeQueryFunction,
   StripesConnectedSource,
   buildUrl,
 } from '@folio/stripes/smart-components';
 
-import filterConfig from './filterConfig';
 import { UserSearch } from '../views';
-import {
-  MAX_RECORDS,
-  USER_TYPES,
-} from '../constants';
+import { MAX_RECORDS } from '../constants';
+import filterConfig from './filterConfig';
 import { buildFilterConfig } from './utils';
 
 const INITIAL_RESULT_COUNT = 30;
 const RESULT_COUNT_INCREMENT = 30;
 
-export const NOT_SHADOW_USER_CQL = `((cql.allRecords=1 NOT type ="") or type<>"${USER_TYPES.SHADOW}")`;
-
```

Comment:
I don't understand why this is being removed. The PR description gives no indication whatsoever.

Reply:
The comment and the request above https://github.com/folio-org/ui-users/pull/2555#issuecomment-1717153370

Reply:
we provide filter to find shadow users, but previously they were excluded from any search results